### ISSUE TITLE
Whip upgrade is wrong as Jester

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#558]
![image](https://user-images.githubusercontent.com/2874150/46467123-e1c37280-c7c4-11e8-8592-840072da9c3b.png)

(As jester, size upgrades do nothing)

**Comment by onetruepurple** [comment]
This can't really be "fixed" without introducing an inconsistency in how items are upgraded, isn't that right?

My suggestion would be to give Whip+ different behavior, or scrap Whip from the Jester pool and give him a similarly working, but different card.

**Comment by TerryCavanagh** [comment]
Yeah, I'll probably make it "burn 3 on a six" or something. It's more interesting that way anyway!
